graylog2-web-interface/src/components/layout/PageContentLayout.tsx

```diff
@@ -68,6 +72,8 @@ PageContentLayout.propTypes = {
 PageContentLayout.defaultProps = {
   children: undefined,
   className: undefined,
+  withFooter: true,
+  NotificationsComponent: undefined,
 };
 
 export default PageContentLayout;
```

Comment:
Just a small suggestion. We could also structure the component like this:

```
const PageContentLayout = ({ children, className, FooterComponent, NotificationsComponent }: Props) => (
  <Container className={className}>
    <NotificationsComponent>
       ....
      <FooterComponent />
    </NotificationsComponent>
  </Container>
);
```

Where the default props are
```
  FooterComponent: Footer,
  NotificationsComponent: StyledWithGlobalAppNotifications,
```

It would unify the component api and we do not need `as={NotificationsComponent}` and `const StyledWithGlobalAppNotifications = styled(WithGlobalAppNotifications)``;`.

Reply:
You're right, this way is more elegant. I've pushed up the changes to it now